Bottom: 1d64c0ad0821d2fbf215518388675ce707d2e01a
Top:    95f948d3797196d87b2dbd9801f08876d3108570
Author: Ian Kent <raven@themaw.net>
Date:   2013-06-11 13:57:14 +0800

autofs-5.0.7 - add changlog entry for coverity fixes

A bunch of changes have been made based on a Covarity report.
Mostly I pust the changes into seperate patches so they can be kept simple.

Changes are:
- fix fcntl return check.
- fix spawn_umount() return check in mount_bind.c:lookup_init().
- fix check mkdir_path() in mount_bind.c:mount_mount().
- fix incorrect var name in test.
- remove debug only code in alarm.c.
- fix inconsistent use of cache lock in handle_packet_missing_direct().
- fix several off by one errors.
- fix memory leak in get_dc_list().
- fix host_addr null reference in add_new_host().
- add null check in read_one().
- add pgrp check in do_spawn().
- fix inconsistent signed usage for __rpc_ping().


---

diff --git a/CHANGELOG b/CHANGELOG
index 48e9806..25179b1 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -49,6 +49,7 @@
 - fix master map bogus keywork match.
 - fix fix map entry duplicate offset detection.
 - probe each nfs version in turn for singleton mounts.
+- add changlog entry for coverity fixes.
 
 25/07/2012 autofs-5.0.7
 =======================
